_config.yml

```diff
@@ -2,9 +2,6 @@
 #theme: jekyll-theme-minimal
 remote_theme: pages-themes/minimal@v0.2.0
 
-# [Conversion]
-markdown: GFM
```

Comment:
Aunque sea el de por defecto, sería mejor poner `kramdown`

Have you tried with kramdown.input = "GMF"? See more about in docs: https://jekyllrb.com/docs/configuration/markdown/